doc/source/examples/constructing_universe.ipynb

```diff
@@ -1,16 +1,15 @@
 {
```

Comment:
<p>Re-pinging this one as a separate comment, I'm not seeing any output here for some reason.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Yes, it's a <code>None</code> because dimensions are set afterwards. I added a <code>print(u.dimensions)</code> to make it more patent. </p>